qmail-send.c

```diff
@@ -1048,7 +1048,7 @@ int c;
  if (birth > recent) n = 0;
  else n = squareroot(recent - birth); /* no need to add fuzz to recent */
  n += chanskip[c];
- return birth + n * n;
+ return birth + (datetime_sec) n * n;
```

Comment:
Why didn't you just change the type of n to datetime_sec 4 lines above?

Reply:
Sure, I guess that makes more sense.